apps/sharedtest/test/unit/dialer/font_size_manager_test.js

```diff
@@ -83,9 +87,19 @@ suite('font size manager', function() {
         setup(function() {
           originalInnerHTML = document.body.innerHTML;
           document.documentElement.style.fontSize = ROOT_FONT_SIZE + 'px';
-          view = document.createElement(viewElement.element);
-          view[viewElement.textField] = 'foobar';
-          document.body.appendChild(view);
+
+          if(viewElement.element === 'bdi') {
+            view = document.createElement('div');
+            document.body.appendChild(view);
+
+            var bdiNode = document.createElement('bdi');
+            bdiNode.textContent = 'foobar';
+            view.appendChild(bdiNode);
+          } else {
+            view = document.createElement(viewElement.element);
+            document.body.appendChild(view);
+            view[viewElement.textField] = 'foobar';
```

Comment:
This is unnecessarily broken up, and it makes the flow confusing. It could be written like this, instead:
```js
var innerEl;
// ...
  view = document.createElement(viewElement.element);
  if (viewElement.childElement) {
    innerEl = document.createElement(viewElement.childElement);
    view.appendChild(innerEl);
  } else {
    innerEl = view;
  }
  innerEl[viewElement.textField] = 'foobar';
  document.body.appendChild(view);
```